src/cli.ts

```
@@ -0,0 +1,156 @@
+typescript
+import { Command } from 'commander';
+import { readFile } from 'node:fs/promises';
+import * as path from 'node:path';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)